public/index.scss

```diff
@@ -4,25 +4,14 @@
  */
 
 .llm-chat-header-icon-wrapper {
-  margin: 0 8px;
-  height: 48px;
   display: flex;
   align-items: center;
+  position: relative;
+  width: 200px;
```

Comment:
Should it be max-width?

Reply:
Do you mean it should be shrink in small screens?

Reply:
I don't know, in terms of responsive I thought max-width would be more suitable. But I do not have much context on this style.

Reply:
Let me leave a comment to UX team. There is no related responsive design for screen width 1200px.